app/views/categories/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="flex flex-col h-full">
+  <div class="flex flex-row w-[vw] bg-[#3778c2] p-3 text-white items-center justify-center">
+    <p class="flex flex-row place-self-center">CATEGORIES</p>
+  </div>
```

Comment:
- [ ] It would be a good thing to include a logout button/ link to give your users a better user experience to not have to navigate back back to login to other accounts.